src/main/java/spoon/reflect/declaration/CtPackage.java

```diff
@@ -149,4 +149,14 @@
 	 * See JLS §7.4.2. Unnamed Packages.
 	 */
 	boolean isUnnamedPackage();
+
+	/**
+	* @return true if the package contains a package-info.java file
+	*/
+	boolean hasPackageInfo();
+
+	/**
+	 * @return true if the package contains a package-info.java and no types
```

Comment:
hard to guess the behavior from the method name. what about removing the test about package-info?

Reply:
Yes. I agree.